build.sh

```diff
@@ -464,18 +464,14 @@ case "$COMMAND" in
 
     "verify-xcframework-evolution-mode")
         export REALM_EXTRA_BUILD_ARGUMENTS="$REALM_EXTRA_BUILD_ARGUMENTS REALM_BUILD_LIBRARY_FOR_DISTRIBUTION=YES"
-        # set the Xcode version to the oldest
-        export REALM_XCODE_VERSION=$REALM_XCODE_OLDEST_VERSION
-        unset REALM_SWIFT_VERSION
-        sh build.sh xcframework osx
-        # copy the xcframework to the testing target
-        rm -rf examples/installation/xcframework-evolution
-        mkdir examples/installation/xcframework-evolution
-        cp -cr build/*.xcframework examples/installation/xcframework-evolution
-        export REALM_XCODE_VERSION=$REALM_XCODE_LATEST_VERSION
         unset REALM_SWIFT_VERSION
+
+        # Build with the oldest supported Xcode version
+        REALM_XCODE_VERSION=$REALM_XCODE_OLDEST_VERSION sh build.sh xcframework osx
+
+        # Try to import the built framework using the newest supported version
         cd examples/installation
-        sh build.sh "test-osx-swift-xcframework"
+        REALM_XCODE_VERSION=$REALM_XCODE_LATEST_VERSION ./build.rb osx xcframework
```

Comment:
The XCFramework test now uses the framework sitting in the `build` directory if it's not asked to fetch a release, so we can reuse it for this test rather than needing a separate target.